velocitas_lib/velocitas_lib.py

```diff
@@ -67,9 +76,21 @@ def get_cache_data() -> dict[str, Any]:
 
 def get_services() -> dict[str, Any]:
     """Return all specified services as Python object."""
+    path = f"{get_package_path()}/runtime.json"
+    variable_value = require_env("runtimeFilePath")
+
+    if variable_value is not None:
+        overwritten_path = Path(variable_value)
+        if not overwritten_path.is_absolute():
+            overwritten_path = Path(get_workspace_dir()).joinpath(overwritten_path)
```

Comment:
this only works under the assumption that the velocitas.json is in the root of the repo or is the workspace_dir always the directory containing the velocitas.json? do we have that requirement somewhere?

Reply:
By CLI design, the `.velocitas.json` always lives in the WORKSPACE_ROOT. It identifies the project as a Velocitas project.

Reply:
Issue in CLI added to make this more clear: https://github.com/eclipse-velocitas/cli/issues/114